### DISCUSSION TITLE
Can we switch tabs usign id in elementor?

### DISCUSSION THREAD

**Discussion by morosmo** [elementor/elementor#22885] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I want to switch tab by id so i can rdirect user to specific page with specific active tab.

### Describe the solution you'd like

Elementor new nested tab feature is awesome. I just love it as its replace the template shortcode and provide faster and easy process. 
I am creating a page where 4 to 5 tabs has there on content. I jsut want to redirect user to page with specific active tab. By default first tab is always active.
I also tried to put id in elementor tab tilte and try with thsi url:
https://website.com/about/#tab_2

but its not working

**Comment by lensbreak** [comment]
+ 1 I was trying to do this as well, currently you need some really ugly javascript placed on page

**Comment by mateitudor** [comment]
I would like this too... it's ridiculous we don't have it by default
